### PR TITLE
Master fix56626

### PR DESCRIPTION
**Pull request by mkrueger**

### CODE REVIEW

File: main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/QuickFixMenuHandler.cs [hunk 1]
```diff
@@ -46,7 +46,7 @@ class QuickFixMenuHandler : CommandHandler
 			quickFixMenu.CommandInfos.Add (new CommandInfo (GettextCatalog.GetString ("Loading..."), false, false), null);
 			info.Add (quickFixMenu);
 			try {
-				var menu = await CodeFixMenuService.CreateFixMenu (editor, ext.GetCurrentFixes (), cancelToken);
+				var menu = await CodeFixMenuService.CreateFixMenu (editor, ext.GetCurrentFixesAsync (cancelToken).Result, cancelToken);
```
Comment (slluis) [inline main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/QuickFixMenuHandler.cs]: You should await the GetCurrentFixesAsync() call.

Comment (mkrueger) [inline main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/QuickFixMenuHandler.cs]: For some reason that doesn't work - didn't find a cause the stack trace didn't say much to me.

Comment (slluis) [inline main/src/addins/MonoDevelop.Refactoring/MonoDevelop.Refactoring/QuickFixMenuHandler.cs]: If it doesn't work, let's fix it.

**Comment by slluis** [comment]
That NotifyChanged() call should not be necessary. I'll see if I can fix it in the command system.

**Comment by slluis** [comment]
I'll fix the notification issue on master.

**Comment by slluis** [comment]
Can you please create a PR for d15-3?

**Pull request merged**